oggm/utils/_funcs.py

```diff
@@ -780,3 +781,96 @@ def shape_factor_adhikari(widths, heights, is_rectangular):
     shape_factors[np.isnan(shape_factors)] = 1.
 
     return shape_factors
+
+
+def cook_rgidf(gi_gdf, region='13', version='60', ids=None,
+               id_suffix=None, save_special_columns=None,
+               assign_col_values=None):
+    """ Cook the user's glacier inventory with the RGI format
+
+    Parameters
+    ----------
+    gi_gdf : :py:geopandas.GeoDataFrame
+        the GeoDataFrame of the user's glacier inventory
+    region : str
+        Glacier RGI region code, only usable when set `confirm_region=False`.
+        The default is '13'.
+    version : str
+        Glacier inventory version code. The default is '60'.
+    ids : list of integer, each element should be a str of length 5
+        Assign special id number to each glacier. The default is None, the id code
+            following the glacier order.
+    id_suffix : str or None
+         Add a suffix to the glacier id. The default is None, no suffix
+    save_special_columns : dict or None
```

Comment:
could this be only one argument (`assign_col_values`) where the right column name in the RGI format is already used? The users can change the name themselves if they want to, I'm not sure this dict here is very useful.

Reply:
OK! `assign_col_values` argument was deleted!